src/main/java/com/develhope/spring/order/dto/OrderMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.develhope.spring.order.dto;
+
+import com.develhope.spring.order.entity.Order;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OrderMapper {
+    public static OrderDTO toDTO(Order order) {
```

Comment:
So che non fa parte della tua pull request, ma segniamoci che è da cambiare OrderDTO in un nome più consono al suo scopo,  ad esempio OrderCreationDTO